tensorflow/python/data/experimental/service/server_lib_test.py

```diff
@@ -18,8 +18,9 @@
 from __future__ import division
 from __future__ import print_function
 
+import tempfile
+import portpicker
```

Comment:
https://github.com/tensorflow/tensorflow/blob/f74cc7a696c66db173b321d51fb05f032652a6c7/tensorflow/python/distribute/multi_worker_test_base.py#L34 has logic for handling `portpicker` not being available, so I think we'll need to do the same thing here.

Reply:
addressed in the commit